src/components/molecules/downloadable-file-button/index.tsx

```diff
@@ -0,0 +1,24 @@
+import FileIcon from "../../fundamentals/icons/file-icon"
+import Button, { ButtonProps } from "../../fundamentals/button"
+import React from "react"
+
+type Props = ButtonProps & {
+  fileName: string
+  fileSize?: string
+}
+
+const DownloadableFileButton = ({ fileName, fileSize, ...buttonProps }: Props) => {
+  return (
+    <Button className="flex justify-start mt-4" {...buttonProps}>
+      <FileIcon size={30}/>
+      <div className="flex flex-col text-left">
+        <span className="overflow-hidden truncate w-4/5">{fileName}</span>
```

Comment:
there is something with the font-weight of the download button. It should be `inter-small-regular` but it looks like it's `inter-small-semibold` from the wrapping button
```suggestion
        <span className="overflow-hidden truncate w-4/5 inter-small-regular">{fileName}</span>
```

Reply:
I ve used base-regular instead to be closer to the fignma design